runtime/kusama/src/lib.rs

```diff
@@ -1139,6 +1139,7 @@ parameter_types! {
 
 impl parachains_ump::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	type XcmTransaction = RuntimeCall;
```

Comment:
I'm not really set on the name `XcmTransaction`, if anyone has a better one, I'm all ears.